### PR TITLE
Added some methods to act on the backend when a piece is eatean howve…

### PR DESCRIPTION
**Pull request by paulopasso**
…r did not yet create the logic for when a piece is eaten. I also created stacks for when every triangle and for when a piece is eaten, or extracted from the board (white and balck respectivel)

### CODE REVIEW

File: game/game.py [hunk 1]
```diff
@@ -36,4 +39,20 @@ def add_piece(self, point, piece: Piece):
         self.points[point].append(piece)
 
     def remove_piece(self, point):
-        return self.points[point].pop() if self.points[point] else None
\ No newline at end of file
+        return self.points[point].pop() if self.points[point] else None
+
```
Comment (RestartDK) [inline game/game.py:43]: Looks good here for eating methods

File: game/dice.py [hunk 2]
```diff
@@ -1,37 +1,90 @@
...
 class Dice:
     def __init__(self):
+        # Load images for the dice faces
         self.faces = [pygame.image.load(f"assets/images/dice-{i}.png") for i in range(1, 7)]
+        self.rolling = False
+        self.roll_start_time = None
+        self.current_face_values = [1, 1, 1, 1]  # Default dice face values
+        self.show_dice = 2  
+        
+        self.rects = [self.faces[0].get_rect() for _ in range(4)]
```
Comment (RestartDK) [inline game/dice.py:15]: Good use of initialising the dice faces

File: game/piece.py [hunk 3]
```diff
@@ -49,3 +63,43 @@ def handle_event(self, event):
             pygame.mouse.set_cursor(pygame.SYSTEM_CURSOR_HAND)
             return True
         return False
+
```
Comment (RestartDK) [inline game/piece.py:66]:
```suggestion
    
```Can you please remove the testing code when I merge it to the main branch?

**Review by RestartDK — CHANGES_REQUESTED** [review]
Ok I looked through the code the dice functionalities work great but could you please submit a new PR and remove the test code before I merge it into the main branch?

File: game/piece.py (hunk 3 above)
Comment (RestartDK) [reply game/piece.py:66]: Wait sorry I did not mean to make a sugestion with code here, just try to remove the testing code here...

**Comment by paulopasso** [comment]
I think that I removed now the test

**Comment by RestartDK** [comment]
I corrected the merge conflicts in your branch I will make a seperate pr to merge it with main

**Review by RestartDK — APPROVED** [review]

**Comment by RestartDK** [comment]
Just to let you know I will delete your branch now on the cloud

**Pull request merged**